source/release-notes/backend.txt

```diff
@@ -13,6 +13,23 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240724:
+
+24 July 2024 Release
+~~~~~~~~~~~~~~~~~~~~
+
+- GraphQL and Static Hosting Deprecation 
+
+  - Apps that use GraphQL or Static Hosting will now display banners in their App Services 
+    UI alerting customers of both the GraphQL and Static Hosting deprecations.
+
+  - The GraphQL template app (web.graphql.todo) is no longer available.
+
+- Fixed bug where disabled Triggers were not resuming properly after being enabled.
+
+- Fixed bug where the Change Origin of Trigger deployments made in the Data Services UI were registered as Admin API
```

Comment:
is "Change Origin" a field somewhere? And this seems like it should be in the general [Atlas Release Notes](https://www.mongodb.com/docs/atlas/release-notes/atlas/), not the App Services ones (since it's referring to triggers created in the Data Services UI, Admin API, and Atlas UI instead of the _App Services_ UI and _App Services_ Admin API)